java/src/com/ibm/streamsx/rest/AbstractStreamingAnalyticsService.java

```diff
@@ -173,8 +179,33 @@ protected void setAuthorization(String authorization) {
     @Override
     public Result<Job, JsonObject> buildAndSubmitJob(File archive, JsonObject jco,
             String buildName) throws IOException {
+    	
```

Comment:
I assume this is temp code and will be removed from the PR before WIP is removed.

Reply:
Correct. This was added by mistake. This is the ssl certificate hack for submitting jobs.